openapi.yaml

```diff
@@ -3177,13 +3171,7 @@ paths:
         content:
           application/json:
             schema:
-              required:
```

Comment:
any reason the required block is removed here? it can be moved into the schema instead

Reply:
Moved it to the schema